src/app/smart-search/page.module.scss

```
@@ -12,7 +12,7 @@
 
   .searchPanel {
     --smart-search-panel-padding: #{pxToRem(8) pxToRem(16) 0 pxToRem(16)};
-    border-right: 2px solid $color-border;
+    border-right: 1px solid $color-border;
```

Stylelint (17.4.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)